src/python_testing/TC_OPCREDS_3_8.py

```diff
@@ -735,7 +733,131 @@ async def test_TC_OPCREDS_3_8(self):
             asserts.assert_true(verify_signature(public_key=noc_public_keys_from_certs["TH2"], message=vendor_id_verification_tbs,
                                 signature=sign_vid_verification_response.signature), "VID Verification Signature must validate using DUT's NOC public key")
 
-        with test_step(22, description="Remove TH2 and TH3's fabric"):
+        # Cover having done UpdateNOC in a Fail-safe, overriding the VVS and VVSC, and then expiring failsafe and seeing the prior values are returned.
+        with test_step(22, description="Start updating fabric under TH2's root with fabric ID 0x2222, node ID 0x22222222, by invoking ArmFailSafe(600s), CSRRequest and UpdateNOC. Do not disarm failsafe, do not execute commissioning complete."):
+            # fabricId is already 0x2222
+            new_th2_nodeId = 0x22222222
+            success = await CommissioningBuildingBlocks.UpdateNOC(
+                devCtrl=th2_dev_ctrl, existingNodeId=th1_dut_node_id + 1, newNodeId=new_th2_nodeId,
```

Comment:
```suggestion
                devCtrl=th2_dev_ctrl, existingNodeId=th2_dut_node_id, newNodeId=new_th2_nodeId,
```

I see `th2_dut_node_id`, `th1_dut_node_id + 1` and `th2_nodeid` being used. Would it make sense to make it consistent? Or they need to be different?

Reply:
Will make it clearer thanks.